configmap/repl-ctl/start/start.sh

```diff
@@ -5,25 +5,28 @@
 # @author  Fabrice Jammes, IN2P3/SLAC
 
 set -e
+# WARN: password are displayed in debug logs
```

Comment:
I have already made a comment regarding passwords in a context of the Replication system